expression/builtin_time_vec_test.go

```diff
@@ -134,7 +134,10 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			geners: []dataGenerator{&rangeInt64Gener{0, 2200}, &rangeInt64Gener{0, 365}},
 		},
 	},
-	ast.MakeTime: {},
+	ast.MakeTime: {
+		{retEvalType: types.ETDuration, childrenTypes: []types.EvalType{types.ETInt, types.ETInt, types.ETReal}},
+		{retEvalType: types.ETDuration, childrenTypes: []types.EvalType{types.ETReal, types.ETReal, types.ETReal}},
```

Comment:
How about limit the range of arguments? For example, we can limit `hour` argument with in [-838*2, +838*2]